molecule/templates/vagrantfile.j2

```diff
@@ -40,6 +40,10 @@ Vagrant.configure('2') do |config|
     vb.linked_clone = true if Vagrant::VERSION =~ /^1.8/
     {%- endif %}
   end
+  # This plugin tries to install packages before the system is registered.  Disable it.
+  if Vagrant.has_plugin?('vagrant-vbguest')
+    config.vbguest.auto_update = false
```

Comment:
Could you parameterize the value (``false`` or ``true``)? If I test against Ubuntu or Centos, it may be useful to be able to use it.